DB/projects.json

```diff
@@ -3668,6 +3668,24 @@
       
     ]
   },
+  {
+    "github_username": "Pro-18",
+    "Social_media": {
+      "gitHub": "https://github.com/Pro-18",
+      "LinkedIn": "https://www.linkedin.com/in/pragati-deshmukh-56080a222/",
+      "Instagram": "#",
+      "YouTube": "#",
+      "Twitter": "#"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Pro-18/wallpaper_viewer",
+        "title": "Wallpaper_Viewer",
+        "description": "The Python Wallpaper_Viewer project is that allows users to view their collection of wallpapers using a GUI built with the Tkinter library",
+        "tech": ["Python"]
+      }
```

Comment:
just add the code to the previous one. You've added projects before. check contributing guidelines for more information.